manual_flagging/initial_flagging.py

```diff
@@ -121,153 +140,373 @@
 df = df.astype(conversion_dict)
 df["ptax_flag_original"].fillna(False, inplace=True)
 
-# Separate res and condo sales based on the indicator column
-df_res = df[df["indicator"] == "res"].reset_index(drop=True)
-df_condo = df[df["indicator"] == "condo"].reset_index(drop=True)
+# Filter to correct tris
+tri_stat_groups = {
+    tri: method
+    for tri, method in inputs["tri_stat_groups"].items()
+    if tri in inputs["run_tri"]
+}
```

Comment:
This narrows down the scope of what we plan to flag. If we only have tri 2 and 3 in the yaml list, this will use that to connect these two tris to the run methodology (`current` or `og_mansueto`) in the `tri_stat_groups` object, which acts as a mapping to tri and run methodology.